Src/main.c

```diff
@@ -1680,6 +1683,10 @@ void runBrushedLoop()
 
 int main(void)
 {
+#ifdef DEBUG_SDI
```

Comment:
we should remove this extra debug code before merge

Reply:
yeah,  all debug related code can be deleted